client/pkg/srv/srv_test.go

```diff
@@ -226,8 +226,8 @@ func TestSRVDiscover(t *testing.T) {
 		},
 		{
 			[]*net.SRV{
-				{Target: "a.example.com", Port: 2480},
-				{Target: "b.example.com", Port: 2480},
+				{Target: "a.example.com.", Port: 2480},
+				{Target: "b.example.com.", Port: 2480},
 				{Target: "c.example.com", Port: 2480},
 			},
 			[]*net.SRV{},
```

Comment:
this unit test is a good example of what looks incorrect at first glance... with a client connection of `[]string{"https://a.example.com:2480", "https://b.example.com:2480", "https://c.example.com:2480"}`, won't local DNS search paths be attempted for these URLs (depending on `ndots` settings)?